src/core/feed_forward.jl

```diff
@@ -353,7 +353,7 @@ function feed_forward_update(sync::Chron,
     !(get_execution_count(to_stage) % sync.to_executions == 0) && return
 
     var_count = get_execution_count(to_stage) ÷ sync.to_executions
-
+    
```

Comment:
Extra whitespace. Please configure the editor to avoid this.

Reply:
ok